### PR TITLE
Support PostgreSQL v16

### PR DESCRIPTION
**Pull request by zwyan0**

### CODE REVIEW

File: data.c [hunk 1]
```diff
@@ -270,7 +270,7 @@ backup_data_file(const char *from_root,
 	BlockNumber			blknum;
 	BlockNumber			segno;
 	size_t				read_len;
-	int					errno_tmp;
+	int					errno_tmp = 0;
```
Comment (mikecaat) [inline data.c:273]: Do we really need errno_tmp? Isn't errno enough?

File: restore.c [hunk 2]
```diff
@@ -1230,9 +1230,9 @@ readTimeLineHistory(TimeLineID targetTLI)
 	elog(DEBUG, "the calculated branch history is as below;");
 	for (i = 0; i < parray_num(result); i++)
 	{
-		pgTimeLine *timeline = parray_get(result, i);
+		pgTimeLine *timeline2 = parray_get(result, i);
```
Comment (mikecaat) [inline restore.c:1233]: Do we need the line? Is `s/timeline2->tli/(parray_get(result, i))->tli/` enough?

**Review by mikecaat — COMMENTED** [review]
Thanks!

File: data.c (hunk 1 above)
Comment (mikecaat) [reply data.c:273]: Sorry, I misunderstood.

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -4,19 +4,19 @@ on:
...
-      - REL_11_STABLE
   pull_request:
     branches:
       - master
+      - REL_16_STABLE
       - REL_15_STABLE
       - REL_14_STABLE
       - REL_13_STABLE
       - REL_12_STABLE
-      - REL_11_STABLE
```
Comment (mikecaat) [inline .github/workflows/build.yml:19]: Will you release pg_rman for v11 in Janually?
I think it's better to remove the line after that.

Additionally you need remove line:49.

Comment (zwyan0) [reply .github/workflows/build.yml]: Understood. I will remove after pg16 support.

**Review by mikecaat — APPROVED** [review]
Thanks.

**Pull request merged**